report/util.go

```diff
@@ -362,6 +362,25 @@ func getDiffCves(previous, current models.ScanResult) models.VulnInfos {
 	return updated
 }
 
+func isNewRelease(current models.VulnInfo, previous models.ScanResult) bool {
+	var preNotFixedYet bool
+	preVinfo, _ := previous.ScannedCves[current.CveID]
```

Comment:
This logic of for loop checks only the last entry of prevInfo.AffectedPackages slice.
so please change as follows to check all elements.

```
_ func isNewRelease(current models.VulnInfo, previous models.ScanResult) bool {
      preVinfo, _ := previous.ScannedCves[current.CveID]
+     pre := map[string]bool{}
      for _, h := range preVinfo.AffectedPackages {
~         pre[h.Name] = h.NotFixedYet
      }

~     cur := map[string]bool{}
      for _, h := range current.AffectedPackages {
~         cur[h.Name] = h.NotFixedYet
      }
~_    return !reflect.DeepEqual(pre, cur)
  }
```